ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/StateStoreLags.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableMap;
+import com.google.errorprone.annotations.Immutable;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Represents the lags associated with a particular state store on a particular host.
+ */
+@Immutable
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class StateStoreLags {
+
+  private final ImmutableMap<Integer, LagInfoEntity> lagByPartition;
+
+  @JsonCreator
+  public StateStoreLags(
+      @JsonProperty("lagByPartition") final Map<Integer, LagInfoEntity> lagByPartition) {
+    this.lagByPartition = ImmutableMap.copyOf(requireNonNull(lagByPartition, "lagByPartition"));
+  }
+
+  public LagInfoEntity getLagByPartition(final int partition) {
+    return lagByPartition.get(partition);
```

Comment:
don't return `null` for unknown partition. 

Either:
* return some kind of _empty_ lag info
* or else encode the fact that the method might not return a value by returning `Optional<LagInfoEntity>`,
* or throw an exception if this is not expected.

Reply:
I changed it to return Optional<LagInfoEntity>